crates/eth_compatibility/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "ink_eth_compatibility"
+version = "3.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
```

Comment:
Why is this file coming back from the dead?

Reply:
I guess some `git merge master` hiccup, I'll bring it back to the dead.